examples/pennylane_advanced_usage.py

```diff
@@ -0,0 +1,140 @@
+r"""
+
+.. _advanced_features:
+
+Advanced Usage
+==============
+
+In the previous three introductory tutorials (:ref:`qubit rotation <qubit_rotation>`, :ref:`Gaussian transformation <gaussian_transformation>`, and :ref:`plugins & hybrid computation <plugins_hybrid>`) we explored the basic concepts of PennyLane, including qubit- and CV-model quantum computations, gradient-based optimization, and the construction of hybrid classical-quantum computations.
```

Comment:
This is now appearing before the "plugins & hybrid computation" tutorial, though it still mentions it as "previous"

Reply:
Fixed